Wrappers/Python/test/test_reconstructors.py

```diff
@@ -466,6 +466,23 @@ def test_weights(self):
         np.testing.assert_allclose(weights,weights_new)
 
 
+    @unittest.skipUnless(has_astra, "ASTRA not installed")
+    def test_set_backend(self):
```

Comment:
I understand that this test is checking that an error is raised if:

1. data with wrong order for the backend is passed
2. an unsupported backend is passed

It's not terribly clear. Maybe a couple of comments will help?